library/src/main/java/com/fuzzproductions/ratingbar/RatingBar.java

```diff
@@ -208,10 +219,21 @@ public boolean onTouch(View v, MotionEvent event) {
                 } else {
                     currentlySelected = i + 1;
                 }
+                if(mRatingBarListener != null && previouslySelected != currentlySelected){
+                    mRatingBarListener.onChangeSelectedStar(
+                            this,
+                            previouslySelected,
+                            currentlySelected
+                    );
+                }
                 updateChildViews();
             }
         }
 
         return true;
     }
+
+    public interface RatingBarListener {
```

Comment:
I noticed that this inner interface differs slightly to its AOSP counterpart. I was thinking that it might be a good idea for this RatingBar class to have API parity with the RatingBar in AOSP. That would be valuable because users would already be familiar with the current API and that would also make migrating between the two RatingBars much easier as it could be just as simple as changing the package name of which RatingBar you wanted to use. What do you think?